operations/boot_orchestration/Clean_Up_After_a_BOS-BOA_Job_is_Completed_or_Cancelled.md

```diff
@@ -40,140 +43,151 @@ When a session is launched, the items below are created:
+    Attempting to prevent CFS from configuring the nodes is a multi-step, tedious process. It may be simpler to allow CFS to configure the nodes. If the nodes are going to be immediately
+    rebooted, then the CFS configuration will be rerun once the nodes have booted, thus undoing any previous configuration. Alternatively, if the nodes are going to be immediately
+    shutdown, then this will remove any need to prevent CFS from configuring the nodes.
 
     Follow the procedures for one of the following use cases:
 
-    -   Configuration has completed and the sessions need to be cleaned up the to reduce clutter:
-        1.  Find the old sessions that needs to be deleted.
+    - Configuration has completed and the sessions need to be cleaned up the to reduce clutter:
+
+        1. (`ncn-mw#`) Find the old sessions that needs to be deleted.
 
             ```bash
             cray cfs sessions list
             ```
 
-        2.  Delete the sessions.
+        1. (`ncn-mw#`) Delete the sessions.
 
             ```bash
             cray cfs sessions delete CFS_SESSION_NAME
             ```
 
-    -   Configuration has completed and the desired state needs to be cleaned up so that configuration does not happen on restart:
-        1.  Unset the desired state for all components affected.
+    - Configuration has completed and the desired state needs to be cleaned up so that configuration does not happen on restart:
 
-            To find the impacted component names (xnames) for the components with the desired configuration matching what was applied:
+        1. (`ncn-mw#`) Unset the desired state for all affected components.
 
-            ```bash
-            cray cfs components list
-            ```
+            1. Find the component names (xnames) for the components with the desired configuration matching what was applied.
 
-            Prevent the configuration from running:
+                ```bash
+                cray cfs components list
+                ```
 
-            ```bash
-            cray cfs components update XNAME --desired-state-commit
-            ```
+            1. Prevent the configuration from running.
 
-            This needs to be done for each component. It is enough to prevent configuration from running, and it does not revert to the previous desired state. The previous desired state has already been overwritten at this point, so if the user is trying to completely revert, they will either need to know and apply the previous desired state manually, or return BOS with the previous template using the configure operation \(which may also trigger a configure operation\).
+                ```bash
+                cray cfs components update XNAME --desired-state-commit
```

Comment:
I don't think this a valid CFS CLI command.

Reply:
It's not.  I assume this should be: cray cfs components update XNAME --desired-config ""

Reply:
```suggestion
                cray cfs components update XNAME --desired-config ""
```